distributions/gcp/v2/cnrm/cluster/cluster.yaml

```diff
@@ -60,3 +60,5 @@ spec:
       name: name-vm # {"$kpt-set":"name-vm"}
     workloadMetadataConfig:
       nodeMetadata: GKE_METADATA_SERVER
+  nodeVersion: '1.16.15-gke.12500'
```

Comment:
I remember it's possible to just specify 1.16, can you confirm?

Reply:
That is a good point, I changed to 1.16 and it is working!